lib/evmone/instructions_storage.cpp

```diff
@@ -6,6 +6,86 @@
 
 namespace evmone::instr::core
 {
+namespace
+{
+struct StorageCostSpec
```

Comment:
We can move this to instruction traits and use in other places (e.g. cost for SLOAD, replacement for `warm_storage_read_cost`.